internal/backends/linuxkms/display/gbmdisplay.rs

```diff
@@ -0,0 +1,128 @@
+// Copyright © SixtyFPS GmbH <info@slint.dev>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-1.1 OR LicenseRef-Slint-commercial
+
+use drm::control::Device;
+use gbm::AsRaw;
+use i_slint_core::platform::PlatformError;
+
+use crate::drmoutput::{DrmOutput, SharedFd};
+
+struct OwnedFramebufferHandle {
+    handle: drm::control::framebuffer::Handle,
+    device: SharedFd,
+}
+
+impl Drop for OwnedFramebufferHandle {
+    fn drop(&mut self) {
+        self.device.destroy_framebuffer(self.handle).ok();
+    }
+}
+
+pub struct GbmDisplay {
+    pub drm_output: DrmOutput,
+    gbm_surface: gbm::Surface<OwnedFramebufferHandle>,
+    gbm_device: gbm::Device<SharedFd>,
+}
+
+impl GbmDisplay {
+    pub fn new(drm_output: DrmOutput) -> Result<GbmDisplay, PlatformError> {
+        //eprintln!("mode {}/{}", width, height);
```

Comment:
there are several of these comments.  I don't mind them too much if it helps future debugging.